timemachine/cpp/src/context.cu

```diff
@@ -76,7 +76,7 @@ Context::multiple_steps(const std::vector<double> &lambda_schedule, int store_du
     int box_buffer_size = x_buffer_size * 3 * 3;
 
     std::vector<double> h_x_buffer(x_buffer_size * N_ * 3);
-
+    cudaStream_t stream = static_cast<cudaStream_t>(0);
```

Comment:
This commit could be dropped. Was from my Local MD PR and had thought it would be more useful. I left it in as I find it more flexible.